.github/workflows/export-master.yaml

```diff
@@ -1,13 +1,9 @@
-name: export
+name: export release prod
```

Comment:
Same comment as [before](https://github.com/kubescape/regolibrary/pull/233/files#r1064092574)

Reply:
@kooomix There are some workarounds mentioned in the [link](https://github.com/orgs/community/discussions/26709#discussioncomment-3253001) in David's comment, but maybe we can just drop this name change?

Reply:
Just reverted the name back to "export".